pkg/server/server.go

```diff
@@ -232,7 +233,6 @@ func buildEtcd(ctx context.Context, cfg *config.Config, logger *zap.Logger, pubA
 	}
 	apiAddr, uerr := url.Parse(apiAddrStr)
 	if uerr != nil {
-		err = errors.WithStack(uerr)
```

Comment:
If you remove this, the returned `err` is nil.

Reply:
Done.